mrtt-ui/src/components/CausesOfDeclineForm.js

```diff
@@ -27,12 +27,12 @@ import {
   SubTitle,
   SubTitle2
 } from '../styles/forms'
-import { ErrorText } from '../styles/typography'
-import ButtonSubmit from './ButtonSubmit'
+import { ButtonSubmit } from './buttons'
```

Comment:
just changed where Submit button lives. I created a ButtonPrimary, but it turns out the org list page doesnt use it. Instead of throwing the code away, which I will def use for the sites list very soon, I am committing it in this ticket and breaking the 'dont merge speculative code' rule.